pages/common/sha1sum.md

```diff
@@ -10,6 +10,14 @@
 
 `sha1sum {{filename1}} {{filename2}}`
 
+- Calculate and save the list of SHA1 checksums to a file:
+
+`sha256sum {{filename1}} {{filename2}} > {{filename.sha1}}`
```

Comment:
Should this be a sha1sum command?

Note: similar examples for other sha*sum commands seem to have the wrong number.